RentallCarsAPI/Tools/CarHelper.cs

```diff
@@ -48,5 +49,14 @@ public List<Car> GetAll()
                 return null;
             }
         }
+
+        public Car GetById(Guid id)
+        {
+            var cars = GetAll();
+            if (cars == null)
+                return null;
+            else
+                return cars.FirstOrDefault(car => car.Id == id);
```

Comment:
No necesitamos el `else`, correcto?
```suggestion
                return cars.FirstOrDefault(car => car.Id == id);
```

Reply:
done